src/tmrc/core/training/train.py

```diff
@@ -69,14 +73,10 @@ def print_model_info(model: torch.nn.Module):
     Args:
         model: Model to print the info of for debugging
     """
-    total_params, non_emb_params = model.get_num_params(), model.get_num_params(
-        non_embedding=True
-    )
-    print(
-        f"Model parameters: {total_params}, non-embedding parameters: {non_emb_params}"
-    )
-    wandb.log({"total_params": total_params, "non_emb_params": non_emb_params})
-    return total_params, non_emb_params
+    total_params = get_num_params(model)
+    print(f"Model parameters: {total_params}")
```

Comment:
Should we add a condition to ensure that models are only printed when `rank=0`?

Reply:
This issue #92 has been added for this. We want to move from `print` to `logging` and in logging you can limit the logging only for `global_rank=0`.